tests/rules/date_comparison.py

```diff
@@ -0,0 +1,6 @@
+#OK
+day1 = "3000-01-15"
+#ERROR: match
```

Comment:
it's a bit ugly, but tests under tests/patterns/ use the #ERROR: syntax to specify where something should match,
but in tests/rules/ we use the #ruleid: xxx syntax.
That's why CI is failing because it finds a match at this line, but there's no #ruleid:  before.
Look at the other test files in this directory for example.